colossalai/graph/__init__.py

```diff
@@ -0,0 +1,5 @@
+import torch
+from packaging import version
+
+assert version.parse(torch.__version__) >= version.parse(
+    '1.10'), "We require at least 1.10 pytorch version when using graph module for a stable torch.fx interface."
```

Comment:
Don't do this as if other modules import this module, then users with pytorch < 1.10 cannot use colossalai at all.

Reply:
ok